docs/principles/operable_principle.md

```diff
@@ -0,0 +1,176 @@
+[⬅️ Principles](../accessibility_principles.md)
+
+# 2. Operable principle
+
+User interface components and navigation must be operable.
+
+## 2.1. Keyboard Accessible
+
+Make all functionality available from a keyboard.
+
+## 2.1.1 Keyboard (Level A)
+
+Make all functionality available from a keyboard.
+
+*This guideline covers point 2.1.1 Keyboard - Level A of the WCAG standard.*
+
+**Techniques**\
+[Android](../platforms/android/techniques_operable_android.md#211-keyboard-level-a), [Flutter](../platforms/flutter/techniques_operable_flutter.md#211-keyboard-level-a)
```

Comment:
Use the heading (level 3) and list instead for better readability;

### Implementation techniques
* [Android](link)
* [Flutter](link)
* [iOS](link)